components/org.wso2.carbon.identity.conditional.auth.functions.common/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/common/utils/Constants.java

```diff
@@ -53,16 +55,19 @@ public class Constants {
      */
     public static class LogConstants {
 
-        public static final String ADAPTIVE_AUTH_SERVICE = "adaptive-auth-service";
+        public static final String ADAPTIVE_AUTH_SERVICE_HTTP_GET = "adaptive-auth-service-http-get";
+        public static final String ADAPTIVE_AUTH_SERVICE_HTTP_POST = "adaptive-auth-service-http-post";
         public static final String FAILED = "FAILED";
 
         /**
          * Define action IDs for diagnostic logs.
          */
         public static class ActionIDs {
 
-            public static final String RECEIVE_TOKEN = "receive-token";
-            public static final String RECEIVE_API_RESPONSE = "receive-api-response";
+            public static final String RECEIVE_TOKEN_HTTP_GET = "receive-token-http-get";
+            public static final String RECEIVE_TOKEN_HTTP_POST = "receive-token-http-post";
+            public static final String RECEIVE_API_RESPONSE_HTTP_GET = "receive-api-response-http-get";
+            public static final String RECEIVE_API_RESPONSE_HTTP_POST = "receive-api-response-http-post";
         }
```

Comment:
If component is getting listed we don't need to qualify the function in the action.
However, we should qualify the function in the message so that it gives the context

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/161/commits/6e8e56035053d28a6459a0a70212190e03aa93fd